operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -3892,6 +4100,10 @@
           ],
           "title": "Tekton Chains CPU Fraction Use",
           "type": "timeseries"
+        }
```

Comment:
any idea why this `Tekton Chains CPU Fraction Use` was repeated @jkhelil  .... perhaps the title is wrong and it needs to changes to reflect what the new panel is ?

Reply:
duplicate, removed now